consensus/istanbul/backend/snapshot.go

```diff
@@ -178,6 +178,19 @@ func (s *Snapshot) apply(headers []*types.Header, gov *governance.Governance, ad
 			return nil, errUnauthorized
 		}
 
+		if number%snap.Epoch == 0 {
+			if len(header.Governance) > 0 {
+				gov.UpdateGovernance(number, header.Governance)
+			}
+			gov.UpdateCurrentGovernance(number)
+			gov.ClearVotes(number)
+
+			// Reload governance values because epoch changed
+			snap.Epoch, snap.Policy, snap.CommitteeSize = getGovernanceValue(gov, number)
+			snap.Votes = make([]governance.GovernanceVote, 0)
+			snap.Tally = make([]governance.GovernanceTallyItem, 0)
```

Comment:
`snap.Votes` and `snap.Tally` are used in `gov.HandleGovernanceVote`. Is it okay to initialize them in here?